src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -190,35 +188,6 @@ private void RunAnalysis(IRequest request, IIssueConsumer consumer, IAnalysisSta
             telemetryManager.LanguageAnalyzed(request.Context.CFamilyLanguage); // different keys for C and C++
         }
 
-        private void HandleMessage(Message message, IRequest request, IIssueConsumer consumer, ref int issueCount)
```

Comment:
Moved to the new class.

There's one change; previously, this method used a `ref issueCount` to count the number of issues that were processed. That's become a read-only property on the new class.